pygmt/src/meca.py

```diff
@@ -305,91 +300,72 @@ def meca(  # noqa: PLR0912, PLR0913, PLR0915
         The component of the seismic moment tensor to plot.
 
         - ``"full"``: the full seismic moment tensor
-        - ``"dc"``: the closest double couple defined from the moment tensor
-          (zero trace and zero determinant)
+        - ``"dc"``: the closest double couple defined from the moment tensor (zero
+          trace and zero determinant)
         - ``"deviatoric"``: deviatoric part of the moment tensor (zero trace)
-    longitude : float, list, or 1-D numpy array
-        Longitude(s) of event location(s). Must be the same length as the
-        number of events. Will override the ``longitude`` values
+    longitude / latitude / depth : float, list, or 1-D numpy array
```

Comment:
With whitespaces, the doc doesn't look correct.
```suggestion
    longitude/latitude/depth : float, list, or 1-D numpy array
```